index.html

```diff
@@ -0,0 +1,156 @@
+        <div class="col-12">
+          <h1 id="who_are_we" class="display-1">Who are we?</h1>
+            <p id="description" class="lead">
+              Authorized <strong>distributors and subcontractors</strong> who provide <strong>gas station maintenance</strong> and <strong>equipment supply.</strong> 
+            </p>
+        </div>
+      </div>
+    </div>
+    <a href="#about" class="animated bounce" id="down">&#x25BE;</a>
+  </section>
+  
+  <section id="about">
+    <div class="container">
+      <div class="row align-items-center crimson-font-color page-head">
+        <div class="col-12 text-center">
+          <p class="title">What we do?</p>
+          <p class="text-muted lead h5 sub-title" style="padding-bottom: 50px;">We provide gas stations with...</p>
+          <div>
+            <p class="about-item">Parts and equiqment installation services</p>
+            <p class="about-item">Tank monitor services</p>
+            <p class="about-item">Maintainance for installed parts</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="container" id="contact">
+      <div class="row align-items-center crimson-font-color page-head">
+        <div class="col-12 text-center">
+          <p class="title">Talk to a human.</p>
+          <p class="text-muted lead h5 sub-title">We are friendly and happy to answer any questions!</p>
+        </div>
+      </div>
+      <div class="row align-items-center">
+        <div class="col-12 col-md-4 text-center item">
+          <h1 class="display-4 title">Call</h1>
+          <a href="tel:9724648661" style="display: block;" class="link">(972) 464-8661</a>
+        </div>
+        <div class="col-12 col-md-4 text-center item">
+          <h1 class="display-4 title">Visit</h1>
+          <a class="link" href="https://www.google.com/maps/place/Gowani+Pump+Supply+%26+LED+Lights/@32.8933603,-96.8934653,17z/data=!3m1!4b1!4m5!3m4!1s0x864c2789219001d5:0x1dbbb31c469499f2!8m2!3d32.8933558!4d-96.8912766" target="_blank" style="display: block;">11311 Harry Hines Blvd Unit 602, Dallas, TX</a>
+        </div>
+        <div class="col-12 col-md-4 text-center item">
+          <h1 class="display-4 title">Email</h1>
+          <a class="link" href="mailto:contact@gowanipump.com" style="display: block;">contact@gowanipump.com</a>
+        </div>
+      </div>
+      <div class="row align-items-center hours-page">
+        <div class="col-12 text-center">
+          <h1 class="display-3 sub-title">We are open...</h1>
+          <dl class="row">
+
+            <dt class="col-sm-6">Monday - Friday</dt>
+            <dd class="col-sm-6">9:00 A.M. - 5:00 P.M.</dd>
+
+            <dt class="col-sm-6">Saturday</dt>
+            <dd class="col-sm-6">9:00 A.M. - 2:00 P.M.</dd>
+
+            <dt class="col-sm-6">Sunday</dt>
+            <dd class="col-sm-6">Closed</dd>
+
+          </dl>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <footer class="footer">
```

Comment:
`<footer>` is already unique, it doesn't need a footer class. You can style the footer tag directly.